### ISSUE TITLE
react-oidc-context doesn't work with webpack 5 module federation

### ISSUE THREAD

**Issue by gorrilla10101** [authts/react-oidc-context#690]
when using react-oidc-context with webpack 5s new module federation it gives a compiler warning and the dev server blocks the UI. 

```
WARNING in shared module react-oidc-context -> /workspaces/GorrillaServices/gorillatroop/host/node_modules/react-oidc-context/dist/esm/react-oidc-context.js
No version specified and unable to automatically determine one. No version in description file (usually package.json). Add version to description file /workspaces/GorrillaServices/gorillatroop/host/node_modules/react-oidc-context/dist/esm/package.json, or manually specify version in shared config.
```

in the webpack config I tried multiple variations of this, which is what the warning says to try but it wouldn't work. 
```
"react-oidc-context": {
          singleton: true,
          requiredVersion: deps["react-oidc-context"],
        }
```

I was able to get it to work by updated the dist/esm/package.json file to include the version. Since its an npm package thats obviously not a long term solution.
```
{
  "type": "module",
  "version": "2.2.2"

}
```

**Comment by giovannefeitosa** [comment]
I have the same issue

**Comment by pamapa** [comment]
I am open to accept a fix for this.

**Comment by barrymichaeldoyle** [comment]
Not sure if this is related but we're facing similar issues:

> Unhandled Promise Rejection: Error: Unsatisfied version 0 from my-app of shared singleton module react-oidc-context

Using `2.2.2`

**Comment by Gitarcitano** [comment]
Hi guys, I've submitted a pull request to fix this warning #936
